From f3feceabf535895f15f410cc2eb78142b5b25208 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 14:01:19 +0800
Subject: [PATCH] Test case3 for Change Fixed PCD under library and driver

Description: Change the default value for a fixed PCD under both a library and a driver (TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf) in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..f75ba98d7b 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -109,6 +109,10 @@
 ################################################################################
 [Components.IA32]
   TestPkg/ResetVector/ResetVector.inf
+  TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf{
+    <PcdsFixedAtBuild>
+    gPcAtChipsetPkgTokenSpaceGuid.PcdIoApicBaseAddress|0x0000001A
+  }
 
   #
   # SEC Phase modules
@@ -118,6 +122,8 @@
     <LibraryClasses>
     NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
 	LockBoxLib|MdeModulePkg\Library\LockBoxNullLib\LockBoxNullLib.inf
+	<PcdsFixedAtBuild>
+    gPcAtChipsetPkgTokenSpaceGuid.PcdIoApicBaseAddress|0x0000001A
   }
 
   #
-- 
2.27.0.windows.1

